### PR TITLE
Solve #1365

### PR DESCRIPTION
**Pull request by cjwyett**
added link to docs in error message.

### CODE REVIEW

File: src/tallies/tally.cpp [hunk 1]
```diff
@@ -240,10 +240,10 @@ score_str_to_int(std::string score_str)
   try {
     MT = std::stoi(score_str);
   } catch (const std::invalid_argument& ex) {
-    throw std::invalid_argument("Invalid tally score \"" + score_str + "\"");
+    throw std::invalid_argument("Invalid tally score \"" + score_str + "\" See the docs for details: https://docs.openmc.org/en/latest/usersguide/tallies.html#scores");
```
Comment (paulromano) [inline src/tallies/tally.cpp:243]:
```suggestion
    throw std::invalid_argument("Invalid tally score \"" + score_str + "\". See the docs "
      "for details: https://docs.openmc.org/en/stable/usersguide/tallies.html#scores");
```

File: src/tallies/tally.cpp [hunk 2]
```diff
@@ -240,10 +240,10 @@ score_str_to_int(std::string score_str)
...
   }
   if (MT < 1)
-    throw std::invalid_argument("Invalid tally score \"" + score_str + "\"");
+    throw std::invalid_argument("Invalid tally score \"" + score_str + "\" See the docs for details: https://docs.openmc.org/en/latest/usersguide/tallies.html#scores");
```
Comment (paulromano) [inline src/tallies/tally.cpp:246]:
```suggestion
    throw std::invalid_argument("Invalid tally score \"" + score_str + "\". See the docs "
      "for details: https://docs.openmc.org/en/stable/usersguide/tallies.html#scores");
```

**Comment by paulromano** [comment]
@cjwyett Looks like you committed one of the suggested changes but not the other

**Comment by cjwyett** [comment]
Ah, so I did, cheers.

**Pull request merged**